### DISCUSSION TITLE
Can someone explain how to browse the local file system via this browser?

### DISCUSSION THREAD

**Discussion by wotuthink** [jlesage/docker-firefox#213] [General]
Can someone explain how can I browse local file system for downloads and uploads? instead of mounting host volume.

**Comment by jlesage** [comment]
With containers, you have to map a directory from the host (local file system) to a directory inside the container.  Do you need more details on how to do this ?

**Reply by wotuthink** [reply]
Yes please. I need more detial on it.

**Reply by jlesage** [reply]
You can look at the quick start (https://github.com/jlesage/docker-firefox#quick-start) for an example.  You should already have a mapping for the `/config` directory, defined by the `-v /docker/appdata/firefox:/config:rw` parameter of the `docker run` command.  If you want another mapping, just add another `-v` parameter to the command.

**Reply by wotuthink** [reply]
okay.
